### ISSUE TITLE
Sidebar toggle is not function in macOS 11.5.1

### ISSUE THREAD

**Issue by khorton** [tgrosinger/advanced-tables-obsidian#102]
**Describe the bug**
The toggle for the Advanced Tables sidebar is not functional.  If the sidebar is present, the toggle will not remove it (situation on computer #1).  If the sidebar is absent, the toggle will not enable the sidebar (situation on computer #2).  Restarting Obsidian and rebooting the computer has no effect.

**To Reproduce**
1. Update to macOS 11.5.1
2. Toggle switch of sidebar in Advanced Tables
3. Observe no change in sidebar status

**Expected behavior**
Sidebar toggle works as it did with earlier OS versions

**Desktop (please complete the following information):**
 - OS: macSO 11.5.1
 - Obsidian Version: v0.12.10
 - Advanced Tables Version: 0.13.0
 
**Additional context**
Sidebar toggles for other plugins still work correctly, so this does not appear to be a global issue with Obsidian, at first glance

**Comment by khorton** [comment]
I'm an idiot.  I'm very new to Obsidian, and didn't fully understand it's interface.  Sidebar toggle is working correctly.  I'll close this.

**Comment by mochsner** [comment]
Interestingly, I've been working on development for a dependency of this project on PopOS (ubuntu derivative) 20.04, and I've noticed the sidebar toggle doesn't work for me when I compile the app locally. 

I **thought** it was an issue with my version of linux, but having seen this issue in my feed, I've just tested on the official plug-in with similar (failing) results  -- clicking the icon when in the table doesn't do anything. 

I don't actively use linux for that much, but it may be worth keeping an eye out for? Pardon me if I'm missing something.
